nbs/10_compute_correlations/05_gtex_v8/py/07_01-gtex-var_pc_log2-clustermatch.py

```diff
@@ -27,19 +27,30 @@
 
 # %% tags=[]
 import pandas as pd
+from tqdm import tqdm
 
 from clustermatch import conf
-from clustermatch.corr import spearman
+from clustermatch.utils import simplify_string
+from clustermatch.corr import clustermatch
 
 # %% [markdown] tags=[]
 # # Settings
 
 # %% tags=[]
-# we don't have gene subsets for recount2
-# GENE_SELECTION_STRATEGY = "var_raw"
+GENE_SELECTION_STRATEGY = "var_pc_log2"
```

Comment:
Why did you decide to change the gene selection strategy?

Reply:
Actually, I never changed it. For some reason I don't remember, git marked these notebooks for GTEx as "renamed" from the deleted notebooks for recount2, and the two deleted lines here are from the recount2 notebook (where we initially computed clustermatch on all genes around ~6700 and no gene selection was needed), but the added line is for genes in GTEx (where we need to do gene selection).

Sorry for that, I guess it complicated all the review.

Reply:
No problem! Just wanted to make sure this is what you wanted (which it sounds like it is).